mobify/platform-scaffold
# WEB-1534: PDP microdata

This PR adds the Product schema structured data to the PDP. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1534
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/857

## Changes
- Updated SFCC and merlins product parsing to not store the currency symbol in the store
- Fixed rendering on PLP and PDP to use global currency value instead of just price string
- Added product schema data to the PDP  (offers/price, name, description, image, URL, ID)

## Todos

## How to test-drive this PR
- cd into the SDK branch WEB-1534__component-support-structured-data
- run `npm run prod:build`
- run `npm link`
- cd back to the scaffold
- run `npm link progressive-web-sdk`
- run `npm start`
- Preview merlins potions
- navigate to a PDP
- copy the markup for the PDP (starting at the '.t-product-details' container)
- paste the markup into the structured data testing tool: https://search.google.com/structured-data/testing-tool
- the data should validate and contain all the necessary properties
- repeat the steps using the SFCC connector and SFCC demo site
